freqtrade/data/history/history_utils.py

```diff
@@ -145,12 +171,12 @@ def refresh_data(*, datadir: Path,
 
 
 def _load_cached_data_for_updating(
-    pair: str,
-    timeframe: str,
-    timerange: Optional[TimeRange],
-    data_handler: IDataHandler,
-    candle_type: CandleType,
-    prepend: bool = False,
+        pair: str,
+        timeframe: str,
+        timerange: Optional[TimeRange],
+        data_handler: IDataHandler,
+        candle_type: CandleType,
```

Comment:
Before looking at this any further, please remove all non-relevant changes from this PR.

Reply:
reverted these changes.